linalg/ginkgo.cpp

```diff
@@ -49,8 +58,12 @@ GinkgoExecutor::GinkgoExecutor(ExecType exec_type)
 #ifdef MFEM_USE_CUDA
             int current_device = 0;
             MFEM_GPU_CHECK(cudaGetDevice(&current_device));
-            executor = gko::CudaExecutor::create(current_device,
-                                                 gko::OmpExecutor::create());
+            if (gko_omp_status == true)
```

Comment:
`if (gko_omp_status)` (and elsewhere in this file)

Reply:
I made this change, but I switched the variable name to `gko_with_omp_support` because I think it reads more clearly in the "short" form of `if (var)`.